tests/acceptance/features/webUIUpload/upload.feature

```diff
@@ -16,6 +16,18 @@ Feature: File Upload
     And file "new-lorem.txt" should be listed on the webUI
     And as "user1" the content of "new-lorem.txt" should be the same as the local "new-lorem.txt"
 
+  @smokeTest
+  Scenario: simple upload of a folder that does not exist before
+    When the user uploads a folder containing following files using the webUI:
+      | lorem.txt     |
+      | new-lorem.txt |
+    Then no message should be displayed on the webUI
+    And the last uploaded folder should be listed on the webUI
+    And as "user1" the last uploaded folder should exist
+    And as "user1" the last uploaded folder should contain the following files inside the sub-folders:
```

Comment:
What to do about `inside the sub-folders` ?
It makes the scenario surprising to read, because we suddenly mention sub-folders.

Reply:
Fixed. It's better not to hide the magic. Fixed above upload steps by mentioning `separate sub-folders`.